tests/test_colors.py

```diff
@@ -10,6 +10,37 @@ def __missing__(self, key):
         return key
 
 
+def test_static_color_methods():
```

Comment:
Thanks for writing tests! Can you move these new tests into its own module like `test_convenience.py`. Also can you invert the asserts so it's `assert expected == test_value` please (such as assert Color('whatever') == Color.method('whatever'))?